metadata_backend/helpers/schemas/datacite.json

```diff
@@ -11,6 +11,10 @@
             "type": "array",
             "title": "Creators",
             "description": "The main researcher(s) involved in producing the data, or the author(s) of the publication.",
+            "required": [
```

Comment:
we might want to add this to `contributors` as well, also the `folder.json` `doiInfo` key needs to be adjusted to reflect this information as well so that any `folder` object gets validated against the correct schema.

`datacite.json` is mostly for rendering the forms in the UI.